specification/ai/Azure.AI.Projects/deployments/models.tsp

```diff
@@ -0,0 +1,79 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../main.tsp";
+import "../common/models.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.Rest;
+
+namespace Azure.AI.Projects;
+
+@doc("Model Deployment Definition")
+@resource("deployments")
+@discriminator("type")
+model Deployment {
+  @doc("The type of the deployment")
+  type: DeploymentType;
+
+  @doc("Name of the deployment")
+  @visibility(Lifecycle.Read)
+  @key("name")
+  name: string;
+}
+
+@doc("Model Deployment Definition")
+model ModelDeployment extends Deployment {
+  @doc("The type of the deployment")
+  type: DeploymentType.ModelDeployment;
+
+  @doc("Publisher-specific name of the deployed model")
+  @visibility(Lifecycle.Read)
+  modelName: string;
+
+  @doc("Publisher-specific version of the deployed model")
+  @visibility(Lifecycle.Read)
+  modelVersion: string;
+
+  @doc("Name of the deployed model's publisher")
+  @visibility(Lifecycle.Read)
+  modelPublisher: string;
+
+  @doc("Capabilities of deployed model")
+  @visibility(Lifecycle.Read)
+  capabilities: Record<string>;
```

Comment:
Enum (Union) for model capabilities (e.g. text input, audio input, image input, text output).